### PR TITLE
[funnyordie] Improve bitrate regex, add _check_formats call for video

### PR DESCRIPTION
**Pull request by savicki**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x ] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x ] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [ x] Improvement
- [ ] New extractor
- [ ] New feature

---

old bitrate regex won't capture all vbrs, see https://regex101.com/r/FXaWjL/2
new version (in this PR): https://regex101.com/r/FXaWjL/1

Also, for some .webm urls server return 403 error
Example: http://www.funnyordie.com/videos/d182501dfe/world-war-g/ , v1200 (.webm)

### CODE REVIEW

File: youtube_dl/extractor/funnyordie.py [hunk 1]
```diff
@@ -51,10 +51,7 @@ def _real_extract(self, url):
 
         formats = []
 
-        formats.extend(self._extract_m3u8_formats(
-            m3u8_url, video_id, 'mp4', 'm3u8_native', m3u8_id='hls', fatal=False))
-
-        bitrates = [int(bitrate) for bitrate in re.findall(r'[,/]v(\d+)[,/]', m3u8_url)]
+        bitrates = [int(bitrate) for bitrate in re.findall(r'v(\d+)[,/]', m3u8_url)]
```
Comment (dstftw) [inline youtube_dl/extractor/funnyordie.py]: This should be fixed with lookahead assertion instead.

File: youtube_dl/extractor/funnyordie.py [hunk 2]
```diff
@@ -65,6 +62,11 @@ def _real_extract(self, url):
                     'vbr': bitrate,
                 })
 
+        self._check_formats(formats, video_id)
```
Comment (dstftw) [inline youtube_dl/extractor/funnyordie.py]: Looks like webm 600 and 2500 are only ever available.

File: youtube_dl/extractor/funnyordie.py (hunk 1 above)
Comment (savicki) [inline youtube_dl/extractor/funnyordie.py]: Done.

**Comment by savicki** [comment]
Thanks for feedback, Sergey

I have updated also md5 sum for 1st test, which get changed after this PR

File: youtube_dl/extractor/funnyordie.py [hunk 3]
```diff
@@ -51,10 +51,7 @@ def _real_extract(self, url):
 
         formats = []
 
-        formats.extend(self._extract_m3u8_formats(
-            m3u8_url, video_id, 'mp4', 'm3u8_native', m3u8_id='hls', fatal=False))
-
-        bitrates = [int(bitrate) for bitrate in re.findall(r'[,/]v(\d+)[,/]', m3u8_url)]
+        bitrates = [int(bitrate) for bitrate in re.findall(r'v(\d+)(?=,|/)', m3u8_url)]
```
Comment (dstftw) [inline youtube_dl/extractor/funnyordie.py]: `r'[,/]v(\d+)(?=[,/])'`.

**Comment by savicki** [comment]
flake8 return only 2 long lines warnings (e501)

**Comment by savicki** [comment]
sorry, there are nine  e501 warnings, not 2

**Pull request closed without merging**